data/human/human missions.txt

```diff
@@ -7453,3 +7453,1836 @@ mission "Adelita 5"
+			`	"Sure, he made some mistakes. No one becomes governor here without out getting a little dirt of their hands. But in the end he refused to give into the company-bosses' demands, and throughout it all he was laboring for the common man." Wynnafirth looks up at you. "You were a good friend to him, it's true, but truth be told, while you may have been his hero, he was ours; he was the real Hero of Rand.`
+			`	"And when he wouldn't budge they finally got him on trumped up charges and threw him to Oblivion to disappear and die." She shakes her head. "There was nothing we could do. We're still in a..." she hesitates, "... a rebuilding phase."`
+			`	She reads the list again. "A few weeks, you say? Well, we already thought he was dead, so if there's a chance we save the Governor then we're in.`
+			`	"We have a few left over explosives and detonators from when this mine was active. That should take care of the bomb he was hoping for." She then unholsters her sidearm and hands it to you.`
+			choice
+				`	(Take it.)`
+
+			`	"And this should take care of his need for a gun. As for the other items on there..." She turns to a torn and faded star chart taped on the rocky cavern wall.`
+			`	Grabbing a small stick leaning from the wall, she points to Rastaban. "Dancer would be the best world to find a survival suit rated for Oblivion's toxic environment. As for medication..." She looks over the chart for a moment, then points to Dabih. "Your best bet will be New Britain. One of the few worlds in the Belt that can produce complex pharmaceuticals."`
+			`	The pointing stick falls on Shaula next. "The best place for a fake ID would be the 'independent' world of Greenrock." She taps her chin before continuing, "It would be tight though getting to all three of these worlds and back to the governor in time. Any of these worlds may also have the other equipment you need, but the quality likely wouldn't be as good."`
+			`	She turns back to you. "I am sorry we couldn't be more help, but hopefully it's enough."`
+			`	After putting the pointing stick back in its place, Wynnafrith steps through the door, indicating for you to follow her.`
+			choice
+				`	(Follow her.)`
+
+			`	Once again in the larger cavern she turns and points at the guard. "Grab our explosive and carefully accompany Captain <last> with it back to his ship."`
+			`	The guard frowns. "But it's our only one..."`
+			`	"Now!" She cuts him off, and watches as he sullenly slumps away.`
+			`	Without looking your way she continues, "It was a pleasure to see you again, Captain <last>, but it's time you were leaving. The governor needs you, and my own glorious revolution needs some attention."`
+			`	You leave the dilapidated mines with a bomb, a pistol, and a plan.`
```

Comment:
Perhaps have a log entry here regarding this mission?

Reply:
Done.